<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formulario</class>
 <widget class="QWidget" name="formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>198</width>
     <height>75</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_nombre"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Edad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_edad"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ciudad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_ciudad"/>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btn_anadir">
    <property name="text">
     <string>AÃ±adir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_borrar">
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
